swagconnect/views.py

```diff
@@ -1,14 +1,18 @@
+from rest_framework.permissions import AllowAny, IsAuthenticated
 from rest_framework.viewsets import ModelViewSet
 
+from .models import SwaggerStorage
 from .serializers import SwaggerStorageSerializer
 
 
 class SwaggerStorageViewSet(ModelViewSet):
     serializer_class = SwaggerStorageSerializer
+    queryset = SwaggerStorage.objects.all()
```

Comment:
Also, add a custom `get_queryset() ' method that returns the objects that the user has access to only.